osu.Game.Rulesets.Osu/Difficulty/Preprocessing/OsuDifficultyHitObject.cs

```diff
@@ -47,9 +42,6 @@ public OsuDifficultyHitObject(HitObject hitObject, HitObject lastLastObject, Hit
             this.lastObject = (OsuHitObject)lastObject;
 
             setDistances();
-
-            // Every strain interval is hard capped at the equivalent of 375 BPM streaming speed as a safety measure
-            StrainTime = Math.Max(50, DeltaTime);
```

Comment:
Shouldn't we have at least *some* capping here just in case? Since strain time is being used in aim as well which isn't being covered by OD nerf

Reply:
I'm removing the cap here knowing it affects aim. Cheeseing becomes way way more prominent when removing speed caps in tapping rather than aim, so ultimately I am at ease leaving aim as is.

Reply:
I see you're doing `max(delta, 1)` a lot anyway though, why not do it in StrainTime?

Reply:
Point taken, reimplemented StrainTime for this exact purpose